assets/css/main.scss

```diff
@@ -47,6 +47,14 @@ a {
 }
 
 .vv-card {
+  .v-card__title {
+    font-size: inherit;
+    font-weight: inherit;
+    letter-spacing: inherit;
+    line-height: inherit;
+    word-break: inherit;
+  }
+
```

Comment:
`.v-card__title` previously didn't define any of these text-related styles. Now it does :-(
For the site to behave as it did previously, it should inherit styles from its parent container styles.